### PR TITLE
Updates to bio and outline

### PR DESCRIPTION
**Pull request by phae**
Updated my details and the outline for my talk. Hope that's all good.

### CODE REVIEW

File: schedule/07_webapps.md [hunk 1]
```diff
@@ -28,14 +28,15 @@ Herman’s Effective JavaScript, and created the TXJS conference (woo!). She has
...
-Whether you know it or not, “UX Designer” is in your job description. Understand more about how you
-can get involved with improving experiences for your site, with a set of practical design principles
-that go beyond interaction and visual design, and can be used by everyone on your team, no matter their
-discipline.
+Building amazing digital services is a dream we all have, but many of us are working for companies that 
+aren't quite with the programme yet. With a few examples from how we're making huge and lasting changes 
+within the UK Government's digital services, I'll describe how you can help steer your products 
+onto better seas through simple culture change tricks mostly involving watching what you say.
+
+Frances is front-end development lead for [GovUK](https://www.gov.uk/) in London, where she is helping to revolutionise
```
Comment (SlexAxton) [inline schedule/07_webapps.md:38]: > revolutionise

lol.

Comment (SlexAxton) [inline schedule/07_webapps.md:38]: (the 's', not the concept)

File: schedule/07_webapps.md [hunk 2]
```diff
@@ -28,14 +28,15 @@ Herman’s Effective JavaScript, and created the TXJS conference (woo!). She has
...
-that go beyond interaction and visual design, and can be used by everyone on your team, no matter their
-discipline.
+Building amazing digital services is a dream we all have, but many of us are working for companies that 
+aren't quite with the programme yet. With a few examples from how we're making huge and lasting changes 
+within the UK Government's digital services, I'll describe how you can help steer your products 
+onto better seas through simple culture change tricks mostly involving watching what you say.
+
+Frances is front-end development lead for [GovUK](https://www.gov.uk/) in London, where she is helping to revolutionise 
+digital services for citizens from the inside out.  Previously, she worked for the BBC on their JavaScript 
+library and for scientific journal publisher NPG.  She blogs at [fberriman](http://fberriman.com/) and
```
Comment (SlexAxton) [inline schedule/07_webapps.md:40]: > .  S

Two spaces. What is this? The 90s?

**Comment by SlexAxton** [comment]
Gotta get my designer to work in branches better, but when he's done destroying and restructuring the site this evening, I'll push these changes up!

Thanks!

File: schedule/07_webapps.md (hunk 2 above)
Comment (phae) [inline schedule/07_webapps.md:40]: I know. I'm like stuck in the dark ages.

File: schedule/07_webapps.md (hunk 1 above)
Comment (phae) [inline schedule/07_webapps.md:38]: Just do what you like. I'm not apologising for being right, dude.

Comment (SlexAxton) [inline schedule/07_webapps.md:38]: > apologising

I see what you did there.

**Pull request merged**